cmd/cstor-pool-mgmt/pool/pool_topology.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pool
+
+import (
+	"encoding/json"
+	"github.com/golang/glog"
+)
+
+// Topology contains the topology strucure of disks used in backend
```

Comment:
Is it good to define these somewhere in apis?
e.g. `pkg/apis/openebs.io/pool/v1alpha1`

This will provide us options to derive functionalities (mock tests, builders, predicates, operations, etc) out of these definitions, just like we do for custom resource schema definitions. These features can be derived from current package as well. However, we will be following a common standard w.r.t packaging.

Reply:
ok.. will move to newly created directory pkg/apis/openebs.io/pool/v1alpha1